native/src/modules/categories/components/CategoryList.js

```diff
@@ -41,20 +54,33 @@ const CategoryThumbnail = styled(Image)`
   margin: 10px;
 `
 
-const VerticalPadding = styled.View`
-  padding: 0 20px;
-`
-
 /**
  * Displays a ContentList which is a list of categories, a caption and a thumbnail
  */
 class CategoryList extends React.Component<PropsType> {
+  getListContent (listContent: ListContentModelType): React.Node {
+    const { theme, language } = this.props
+    const cacheDictionary = mapValues(listContent.files, (file: PageResourceCacheEntryStateType) => {
+      return file.filePath.startsWith(RESOURCE_CACHE_DIR_PATH)
+        ? file.filePath.replace(RESOURCE_CACHE_DIR_PATH, listContent.resourceCacheUrl)
```

Comment:
Maybe we should rather put the right values directly in the cache dictionary. Perfectly fine though in this PR.

Reply:
@klinzo do you want to create a task for this? Does this require changes in a lot of places?

Reply:
I don't think it is worth to create a task for this. I mean we have so many tasks already, this can be refactored once we have a third occurrence of this code or we change something here anyway.

Reply:
I'm oke with not changing it now :D but having many tasks should not be an argument :D